sql/src/test/java/io/crate/integrationtests/PartitionedTableConcurrentIntegrationTest.java

```diff
@@ -315,4 +315,43 @@ public void testDeletePartitionWhileInsertingData() throws Exception {
     public void testDeletePartitionWhileBulkInsertingData() throws Exception {
         deletePartitionWhileInsertingData(true);
     }
+
+    private static Map createColumnMap(int numCols, String prefix) {
+        Map<String, Object> map = new HashMap<>(numCols);
+        for (int i = 0; i < numCols; i++) {
+            map.put(String.format("%s_col_%d", prefix, i), i);
+        }
+        return map;
+    }
+
+
+    @Test
+    public void testInsertIntoDynamicObjectColumn() throws Exception {
```

Comment:
Please extend the method name or add a comment explaining that this is testing that the template update works & that all columns will end up in the metadata/tableInfo